reflex/.templates/apps/blank/code/blank.py

```diff
@@ -11,24 +11,31 @@
 class State(rx.State):
     """The app state."""
 
+    ...
+
 
 def index() -> rx.Component:
-    return rx.center(
-        rx.theme_panel(),
+    # Welcome Page (Index)
+    return rx.container(
+        rx.color_mode.button(position="top-right"),
         rx.vstack(
             rx.heading("Welcome to Reflex!", size="9"),
-            rx.text("Get started by editing ", rx.code(filename)),
-            rx.button(
+            rx.text("Get started by editing ", rx.code(filename), size="5"),
+            rx.link(
                 "Check out our docs!",
-                on_click=lambda: rx.redirect(docs_url),
-                size="4",
+                href=docs_url,
+                is_external=True,
+                style={
```

Comment:
I feel we shouldn't have styles if not needed (just use the default) - or if we do, we should pass them as keyword args rather than a dict.